src/build123d.egg-info/PKG-INFO

```diff
@@ -19,6 +19,8 @@ Build123d could be considered as an evolution of [CadQuery](https://cadquery.rea
 
 The documentation for **build123d** can found at [readthedocs](https://build123d.readthedocs.io/en/latest/index.html).
 
+There is a [***Discord***](https://discord.com/invite/Bj9AQPsCfx) server (shared with CadQuery) where you ask for help in the development channel.
+
 To install **build123d** from github:
 ```
 python3 -m pip install git+https://github.com/gumyr/build123d.git@pyproject.toml
```

Comment:
The install can be shortened: `python3 -m pip install git+https://github.com/gumyr/build123d`

Reply:
As the .egg-info isn't in GitHub - I guess this change isn't really useful anymore.